R/utils.R

```diff
@@ -197,6 +198,10 @@ enw_example <- function(type = c(
 #'   }
 #' )
 coerce_date <- function(dates) {
+  if (is.null(dates)) {
+    return(data.table::as.IDate(.Date(numeric())))
+  }
+
```

Comment:
```suggestion
  if (is.null(dates)) {
    return(data.table::as.IDate(c()))
  }

```

Reply:
maybe `numeric` or `integer` would be faster allocation, but `c()` i think is clearly to people as "empty vector".

Reply:
does this work though?

Reply:
yes, I tried it - returns an empty IDate vector (unlike providing no argument for x at all, which would be the ideal version).